`yay -Qu` fails with error

#### Affected Version
yay v9.4.3 - libalpm v12.0.1

#### Issue
`yay -Qu` fails with error

#### Steps to reproduce
1. run `yay -Qu`

#### Output
`yay -Qu`

```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/Jguer/yay/v9/pkg/stringset.StringSet.Set(...)
	pkg/stringset/stringset.go:27
main.upList(0xc000101d70, 0x391, 0x400, 0xc0002b8200, 0x10, 0x10, 0xc0002ec000, 0x391, 0x400)
	upgrade.go:133 +0x1095
main.printUpdateList(0xc000082300, 0xc000101e58, 0x2)
	print.go:404 +0x12b
main.handleQuery(0xa17ab1, 0x1)
	cmd.go:181 +0x7c
main.handleCmd(0x0, 0x0)
	cmd.go:155 +0x1e9
main.main()
	main.go:231 +0xeb
```

`yay -Pg`

```
{
	"aururl": "https://aur.archlinux.org",
	"buildDir": "/home/enigma/.cache/yay",
	"absdir": "/home/enigma/.cache/yay/abs",
	"editor": "",
	"editorflags": "",
	"makepkgbin": "makepkg",
	"makepkgconf": "",
	"pacmanbin": "pacman",
	"pacmanconf": "/etc/pacman.conf",
	"redownload": "no",
	"rebuild": "no",
	"batchinstall": false,
	"answerclean": "",
	"answerdiff": "",
	"answeredit": "",
	"answerupgrade": "",
	"gitbin": "git",
	"gpgbin": "gpg",
	"gpgflags": "",
	"mflags": "",
	"sortby": "votes",
	"searchby": "name-desc",
	"gitflags": "",
	"removemake": "ask",
	"sudobin": "sudo",
	"sudoflags": "",
	"requestsplitn": 150,
	"sortmode": 0,
	"completionrefreshtime": 7,
	"sudoloop": false,
	"timeupdate": false,
	"devel": false,
	"cleanAfter": false,
	"provides": true,
	"pgpfetch": true,
	"upgrademenu": true,
	"cleanmenu": true,
	"diffmenu": true,
	"editmenu": false,
	"combinedupgrade": false,
	"useask": false
}
```

--- a/print.go
+++ b/print.go
@@ -377,7 +377,7 @@ func localStatistics() error {
 //TODO: Make it less hacky
 func printNumberOfUpdates() error {
 	//todo
-	warnings := &aurWarnings{}
+	warnings := makeWarnings()
 	old := os.Stdout // keep backup of the real stdout
 	os.Stdout = nil
 	aurUp, repoUp, err := upList(warnings)
@@ -393,7 +393,7 @@ func printNumberOfUpdates() error {
 //TODO: Make it less hacky
 func printUpdateList(parser *arguments) error {
 	targets := stringset.FromSlice(parser.targets)
-	warnings := &aurWarnings{}
+	warnings := makeWarnings()
 	old := os.Stdout // keep backup of the real stdout
 	os.Stdout = nil
 	_, _, localNames, remoteNames, err := filterPackages()

